src/h5web/App.test.tsx

```diff
@@ -126,9 +126,115 @@ test('navigate groups in explorer', async () => {
 
   expect(queryByRole('treeitem', { name: '1_cormap' })).not.toBeInTheDocument();
 
-  // `waitFor` avoids warning due to async work occuring after completion of test
+  // `waitFor` avoids warning due to async work occurring after completion of test
   await waitFor(() => {
     expect(groupBtn).toHaveAttribute('aria-selected', 'true');
     expect(groupBtn).toHaveAttribute('aria-expanded', 'false');
   });
 });
+
+test('select a scalar dataset and display the value', async () => {
+  const { findByRole, getByRole, getByText } = renderApp();
+
+  // Select the scalar dataset entry_0000/1_cormap/version
+  fireEvent.click(await findByRole('treeitem', { name: 'entry_0000' }));
+  fireEvent.click(getByRole('treeitem', { name: '1_cormap' }));
+  const datasetBtn = getByRole('treeitem', { name: 'version' });
+  expect(datasetBtn).toBeVisible();
+  fireEvent.click(datasetBtn);
+  expect(datasetBtn).toHaveAttribute('aria-selected', 'true');
+  expect(datasetBtn).toHaveAttribute('aria-expanded', 'false'); // A dataset cannot be expanded
+
+  const scalarTab = await findByRole('tab', { name: 'Scalar' });
+  expect(scalarTab).toBeVisible();
+  expect(scalarTab).toHaveAttribute('aria-selected', 'true');
+
+  const datasetValue = getByText('0.7.1');
+  expect(datasetValue).toBeVisible();
+});
+
+test('switch visualisations for a 1D dataset', async () => {
+  const { findByRole, getByRole } = renderApp();
+
+  // Select the 1D dataset entry_0000/1_cormap/to_merge
+  fireEvent.click(await findByRole('treeitem', { name: 'entry_0000' }));
+  fireEvent.click(getByRole('treeitem', { name: '1_cormap' }));
+  const datasetBtn = getByRole('treeitem', { name: 'to_merge' });
+  expect(datasetBtn).toBeVisible();
+  fireEvent.click(datasetBtn);
+
+  const visSelector = await findByRole('tablist', { name: 'Visualization' });
+  expect(visSelector).toBeVisible();
```

Comment:
I don't know that these two lines are worth having. You're already testing the tabs below.